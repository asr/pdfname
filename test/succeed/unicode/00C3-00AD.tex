\documentclass{article}
\usepackage{hyperref}

% Unicode `Ã\x00AD` (U+00C3 and U+00AD).

\hypersetup{%
  pdfauthor={MarÃ­n},%
  pdftitle={MarÃ­n},%
  pdfcreationdate={D:2017},%
}

\begin{document}
pdfname test.
\end{document}
