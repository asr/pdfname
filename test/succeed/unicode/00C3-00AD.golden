ret > ExitSuccess
out > The full path name will be /tmp/marin-no-year.marin.pdf
out >
